tools/topology/topology1/platform/intel/intel-boards.m4

```diff
@@ -0,0 +1,34 @@
+#
+# Machine driver differentiation for dai link ID
+#
+
+# variable that need to be defined in upper m4
+ifdef(`PLATFORM',`',`fatal_error(note: Need to define platform idendifier for intel-boards)')
+ifdef(`BOARD',`',`fatal_error(note: Need to define board idendifier for intel-boards)')
```

Comment:
typo: identifier

Reply:
also typo in commit title: s/machind/machine/

Reply:
fixed